yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -664,24 +664,64 @@ func (yb *TargetYugabyteDB) IsNonRetryableCopyError(err error) bool {
 	return err != nil && utils.InsensitiveSliceContains(NonRetryCopyErrors, err.Error())
 }
 
+/*
+TODO(future): figure out the sql error codes for prepared statements which have become invalid
+and needs to be prepared again
+*/
 func (yb *TargetYugabyteDB) ExecuteBatch(migrationUUID uuid.UUID, batch EventBatch) error {
+	log.Infof("executing batch of %d events", len(batch.Events))
+	ybBatch := pgx.Batch{}
+	stmtToPrepare := make(map[string]string)
+	// processing batch events to convert into prepared or unprepared statements based on Op type
+	for i := 0; i < len(batch.Events); i++ {
+		event := batch.Events[i]
+		if event.Op == "u" {
+			stmt := event.GetSQLStmt(yb.tconf.Schema)
+			ybBatch.Queue(stmt)
```

Comment:
Hmm... that means you're not using prepared statements for the update events?

Reply:
yes